assets/scss/common/top-bar.scss

```diff
@@ -388,6 +375,10 @@ $topbar-dropdown-arrows: false !default; //Set false to remove the \00bb >> text
           & > .dropdown {
             @include topbar-show-dropdown();
             width: 100%;
+            top: 0;
+            &:before {
+              display: none;
```

Comment:
@magdamaciaszczyk What does this do? I'm thinking that because there is no `content:` declared here that there is nothing to hide?